ch_integration_tests/run_tests.sh

```diff
@@ -57,5 +57,6 @@ if [ -d "$LINUX_CUSTOM_DIR" ]; then
 fi
 
 pushd ch_integration_tests
-cargo test -- --test-threads=1
+#cargo test -- --test-threads=1
+cargo test -- test_bridge_networking --nocapture RUST_BACKTRACE=1
```

Comment:
This is a leftover from debugging I think.

Reply:
oh yes :) let me fix.